packages/core/services/AnnotationService/DatabaseAnnotationService/index.ts

```diff
@@ -61,27 +61,30 @@ export default class DatabaseAnnotationService implements AnnotationService {
         const filtersByAnnotation = filters.reduce(
             (map, filter) => ({
                 ...map,
-                [filter.name]: map[filter.name]
-                    ? [...map[filter.name], filter.value]
-                    : [filter.value],
+                [filter.name]: map[filter.name] ? [...map[filter.name], filter] : [filter],
             }),
-            {} as { [name: string]: (string | null)[] }
+            {} as { [name: string]: FileFilter[] }
         );
 
         hierarchy
             // Map before filter because index is important to map to the path
             .forEach((annotation, index) => {
                 if (!filtersByAnnotation[annotation]) {
-                    filtersByAnnotation[annotation] = [index < path.length ? path[index] : null];
+                    filtersByAnnotation[annotation] = [
+                        new FileFilter(annotation, index < path.length ? path[index] : null),
```

Comment:
Will this result in creating the appropriate `type` of `FileFilter`? Ex. the "else" parse of the ternary here returns a `null` which is essentially trying to create an `IncludeFilter`. If the appropriate `type` isn't present in the `FileFilter`, won't the `toQueryString` return the wrong string?